services/ui/display/screen_manager_ozone_internal.cc

```diff
@@ -320,7 +320,8 @@ void ScreenManagerOzoneInternal::PostDisplayConfigurationChange() {
     }
   }
 
-  touch_transform_controller_->UpdateTouchTransforms();
+  if (touch_transform_controller_)
```

Comment:
why do you need to change this? This is mus/mash related.

Reply:
If you look at the sequence described at the previous comment, the crash happened with 'views_mus_unittests', it's started at https://cs.chromium.org/chromium/src/services/ui/display/screen_manager_ozone_internal.cc?q=ScreenManagerOzoneInternal::Init()&sq=package:chromium&g=0&l=158 and |touch_transform_controller_| wasn't created yet. So, accessing without checking caused crash.

Reply:
ok